src/main/java/duke/Command.java

```diff
@@ -0,0 +1,130 @@
+package duke;
+import duke.exception.DukeException;
+import duke.tasks.*;
+
+import java.time.format.DateTimeParseException;
+
+
+public class Command {
+
+    private String[] cmd;
+
+    private boolean isExit = false;
+
+    public Command(String[] cmd) {
+        this.cmd = cmd;
+    }
+
+    /**
+     * Determines the Nature of the command to be executed
+     * @param taskList the TaskList which is active
+     * @param ui the user interface
+     * @param storage The object used to write and save state of Tasklist
+     * @throws DukeException
+     */
+    public void execute(TaskList taskList, Ui ui, Storage storage) throws DukeException {
+        switch(cmd[0]) {
+            case "list" :
+                ui.printList(taskList.getList());
+                break;
+            case "mark" :
+                taskList.mark(cmd[1]);
+                break;
+            case "unmark" :
+                taskList.unmark(cmd[1]);
+                break;
+            case "find" :
+                ui.showSearchResults(taskList.searchKeyword(cmd[1]));
+                break;
+            //Event creation
+            case "todo":
+                createTodo(cmd, taskList, ui);
+                break;
+            case "deadline":
+                createDeadline(cmd, taskList, ui);
+                break;
+            case "event":
+                createEvent(cmd, taskList, ui);
+                break;
+            case "delete":
+                taskList.removeTask(cmd[1]);
+                break;
+            case "bye" :
+                isExit = true;
+                break;
+            default:
+                throw new DukeException("I'm not sure what that means :((");
```

Comment:
The indentations are off for your switch cases. The case should be inline with the switch function.

Reply:
@KeithCZW Thanks for the feedback! do u have a link which shows the case and switch being inline? most of the examples I have googled have shown otherwise.

Reply:
https://se-education.org/guides/conventions/java/intermediate.html#layout

Under layout section, switch and cases are aligned.

Reply:
Thanks!